<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChessAi</class>
 <widget class="QMainWindow" name="ChessAi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChessAi - Stockfish Engine</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="labelTitle">
      <property name="text">
       <string>Chess with Stockfish Engine</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>300</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="labelMove">
        <property name="text">
         <string>Enter Move (UCI format):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditMove">
        <property name="placeholderText">
         <string>e.g., e2e4</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnMakeMove">
        <property name="text">
         <string>Make Move</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="btnNewGame">
      <property name="text">
       <string>New Game</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelHelp">
      <property name="text">
       <string>Instructions: Click "New Game" to start. Enter your moves in UCI format (e.g., e2e4 for pawn from e2 to e4). The engine will respond with its best move.</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
